composer.json

```diff
@@ -11,14 +11,17 @@
 	],
 	"require": {
 		"php": ">=7.1",
+		"altis/consent-api": "^0.9",
```

Comment:
Just occurred to me but this should be a dependency of the consent plugin rather than a direct dependency here

Reply:
I wasn't sure how that would work since it needs to be in the main `vendor` directory rather than the consent plugin's `vendor` directory. I assume the [installation handling](https://github.com/humanmade/altis-core/pull/92/files#diff-093e57f43775d5bb8a825af36fbd068bR46) that's done in `class-installer` would handle that even if it's further down the tree?

Reply:
Yep correct, that list of packages is the complete list once Composer has figured out the entire tree.

Reply:
Cool cool. I'll update that. 👍

Reply:
https://github.com/humanmade/altis-consent/pull/21